sqlmesh/dbt/builtin.py

```diff
@@ -246,6 +250,18 @@ def _try_literal_eval(value: str) -> t.Any:
         return value
 
 
+def _dbt_macros_registry() -> JinjaMacroRegistry:
+    site_packages = next(p for p in sys.path if "site-packages" in p)
```

Comment:
is this right? are you sure it's always the first one?

Reply:
As far as I can tell, yes. Would you prefer i look at all entries that include "site-packages"?

Reply:
Actually, I'll look for "site-packages/dbt".